### DISCUSSION TITLE
路由懒加载失败，需要开放一个加载失败的事件

### DISCUSSION THREAD

**Discussion by sweet7788** [umijs/umi#7510] [Ideas]
1. 用户打开网页长时间未关闭。
2. 服务端打包重新更新后，资源文件名称变了。 此时用户用之前的网页打开一个未加载过的模块，就加载失败了。
需要开放一个加载失败的事件， 开发者可以在这个事件中帮助用户重新刷新浏览器。这样看上去不像是程序故障。

**Comment by hcsheng1998** [comment]
同感, 希望官方能够尽快解决
